### PR TITLE
solution

### PR DESCRIPTION
**Pull request by savchuksavchuk**
[DEMO LINK](https://savchuksavchuk.github.io/react_goods-selector/)

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -15,71 +15,87 @@ export const goods = [
...
+  state = {
+    selectedGood: 'Jam',
+  };
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  setGood = (event: React.MouseEvent<HTMLButtonElement>) => {
+    const goodName = event.currentTarget.closest('tr')
+      ?.querySelector('.is-vcentered')
+      ?.textContent;
```
Comment (Bohdan-Maksymiuk) [inline src/App.tsx:26]: just pass goodName as an argument into callback

File: src/App.tsx [hunk 2]
```diff
@@ -15,71 +15,87 @@ export const goods = [
...
+    const { selectedGood } = this.state;
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Jam
-          </td>
-        </tr>
+    const goodIsSelectedMessage = (
+      <h1 className="title is-flex is-align-items-center">
+        {selectedGood}
+        {' is selected'}
```
Comment (Bohdan-Maksymiuk) [inline src/App.tsx:45]: use interpolation here

File: src/App.tsx [hunk 3]
```diff
@@ -15,71 +15,87 @@ export const goods = [
...
+        <table className="table">
+          <tbody>
+
+            {goods.map(good => (
+              <tr
+                data-cy="Good"
+                key={good}
+                className={selectedGood === good
+                  ? 'has-background-success-light'
+                  : ''}
```
Comment (Bohdan-Maksymiuk) [inline src/App.tsx:76]: use [classnames](https://www.npmjs.com/package/classnames) library to add classes conditionally

File: src/App.tsx [hunk 4]
```diff
@@ -15,71 +15,87 @@ export const goods = [
...
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+        {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
```
Comment (Bohdan-Maksymiuk) [inline src/App.tsx:47]: try to avoid disabling eslint rules - lets fix them

**Review by Bohdan-Maksymiuk — CHANGES_REQUESTED** [review]

File: src/App.tsx (hunk 4 above)
Comment (savchuksavchuk) [reply src/App.tsx]: The only way I found to avoid this eslint rule disabling is to put some invisible content in the button tag

**Review by loralevitska — APPROVED** [review]
good job!

**Pull request closed without merging**